<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
border: none;
}

#centralwidget{
background-color: white;
}

/* Header Container */
#header{
background-color: rgb(68, 62, 255);
border: 1px solid black
}

/* NavBar Container */
#homeButton, #pomoFocusButton, #libraryButton, #reportsButton, #settingsButton, #logoutButton{
padding: 15px;
font-size: 20px;
}

#leftContainer{
background-color: rgb(68, 62, 255);
border: 1px solid black
}

#profilePictureLabel{
border:1px solid black;
}

/* Body Container */

/* PomoFocus Page */

/* Pomo Container */

#pomoContainer{
border: 2px solid black;
}

#tasksHeader{
border-bottom: 1px solid black;
}

#clockContainer{
background-color: rgb(159, 51, 154);
}

#startStopButton{
color: rgb(159, 51, 154);
background-color: white;
padding: 10px;
}

#pomodoroButton, #shortBreakButton, #longBreakButton{
background-color : rgb(186, 89, 199);
padding: 10px
}

#subjectsComboBox{
border: 1px solid black;
}

/* Tasks Container */
#tasksContainer{
border: 2px solid black;
}

#addTaskButton{
background-color: rgb(186, 89, 199);
padding: 15px
}


#label_2{
background-color: orange;
padding: 10px;
font-size: 30px
}






</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftContainer" native="true">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Stay Focused by mo3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>20</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="profilePictureLabel">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>150</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>150</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../me.jpg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Welcome, Moadh</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>50</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="homeButton">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#homeButton{
background-color: white;
padding: 15px;
font-size: 20px;
border-top-left-radius: 30px
}
</string>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pomoFocusButton">
               <property name="text">
                <string>PomoFocus</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="libraryButton">
               <property name="text">
                <string>Library</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="reportsButton">
               <property name="text">
                <string>Reports</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="settingsButton">
               <property name="text">
                <string>Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="logoutButton">
               <property name="text">
                <string>Logout</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightContainer" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="header" native="true"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="pagesStackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="homePage">
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>290</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>home</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pomoFocusPage">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="pomoContainer" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_13" stretch="4,1">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="clockContainer" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,10,0">
                    <item>
                     <widget class="QFrame" name="frame_3">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QPushButton" name="pomodoroButton">
                         <property name="font">
                          <font>
                           <pointsize>15</pointsize>
                           <weight>50</weight>
                           <bold>false</bold>
                           <underline>false</underline>
                          </font>
                         </property>
                         <property name="text">
                          <string>Pomodoro</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="shortBreakButton">
                         <property name="font">
                          <font>
                           <pointsize>15</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Short Break</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="longBreakButton">
                         <property name="font">
                          <font>
                           <pointsize>15</pointsize>
                          </font>
                         </property>
                         <property name="text">
                          <string>Long Break</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QFrame" name="frame_4">
                      <property name="frameShape">
                       <enum>QFrame::StyledPanel</enum>
                      </property>
                      <property name="frameShadow">
                       <enum>QFrame::Raised</enum>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                       <item>
                        <widget class="QLabel" name="timerLabel">
                         <property name="font">
                          <font>
                           <pointsize>70</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <property name="text">
                          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QPushButton" name="startStopButton">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>30</pointsize>
                        <weight>50</weight>
                        <bold>false</bold>
                       </font>
                      </property>
                      <property name="styleSheet">
                       <string notr="true"/>
                      </property>
                      <property name="text">
                       <string>Start</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="font">
                       <font>
                        <pointsize>12</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Choose a subject to focus on (optional) :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="subjectsComboBox">
                      <property name="minimumSize">
                       <size>
                        <width>150</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>15</pointsize>
                        <weight>50</weight>
                        <italic>false</italic>
                        <bold>false</bold>
                        <underline>false</underline>
                       </font>
                      </property>
                      <property name="editable">
                       <bool>false</bool>
                      </property>
                      <item>
                       <property name="text">
                        <string>Math</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Physics</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Science</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Philo</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Francais</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>English</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="tasksContainer" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QFrame" name="tasksHeader">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="font">
                       <font>
                        <pointsize>15</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Tasks</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_5">
                      <property name="styleSheet">
                       <string notr="true">background-color: brown</string>
                      </property>
                      <property name="text">
                       <string>settings</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_5">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_12">
                    <item>
                     <widget class="QPushButton" name="addTaskButton">
                      <property name="font">
                       <font>
                        <pointsize>15</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string>Add Task</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QScrollArea" name="scrollArea">
                   <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="widgetResizable">
                    <bool>true</bool>
                   </property>
                   <widget class="QWidget" name="scrollAreaWidgetContents">
                    <property name="geometry">
                     <rect>
                      <x>0</x>
                      <y>0</y>
                      <width>297</width>
                      <height>496</height>
                     </rect>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <item alignment="Qt::AlignTop">
                      <widget class="QFrame" name="frame_6">
                       <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                       </property>
                       <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_14">
                        <item>
                         <widget class="QLabel" name="label_2">
                          <property name="styleSheet">
                           <string notr="true"/>
                          </property>
                          <property name="text">
                           <string>eat shit</string>
                          </property>
                          <property name="wordWrap">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="libraryPage">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>330</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>library</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="reportsPage">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>340</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>reports</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="settingsPage">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>320</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>settings</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
